tests/integration/ceph_cosi_test.go

```diff
@@ -138,9 +138,10 @@ func testCOSIDriver(s *suite.Suite, helper *clients.TestClient, k8sh *utils.K8sH
 	})
 
 	t.Run("delete COSI Controller", func(t *testing.T) {
-		_, err := k8sh.Kubectl("delete", "-k", "github.com/kubernetes-sigs/container-object-storage-interface-api")
-		assert.NoError(t, err, "failed to create COSI CRDs")
-		_, err = k8sh.Kubectl("delete", "-k", "github.com/kubernetes-sigs/container-object-storage-interface-controller")
-		assert.NoError(t, err, "failed to create COSI controller")
+		// time.Sleep(10 * time.Second)
```

Comment:
What errors do we get here if it is deleted? This is the issue @thotz needs to look into? I'd suggest we revert this change, and merge the changes with the increased timeouts. Then @thotz can separately look at the COSI tests.